rules/integrations/aws/initial_access_console_login_root.toml

```
@@ -21,13 +21,55 @@ interval = "10m"
 language = "kuery"
 license = "Elastic License v2"
 name = "AWS Management Console Root Login"
-note = """## Setup
+note = """## Triage and analysis
+
+### Investigating AWS Management Console Root Login
+
+The AWS root account is the one identity that has complete access to all AWS services and resources in the account,
+which is created when the AWS account is created. AWS strongly recommends that you do not use the root user for your
+everyday tasks, even the administrative ones. Instead, adhere to the best practice of using the root user only to create
+your first IAM user. Then securely lock away the root user credentials and use them to perform only a few account and
+service management tasks. AWS provides a [list of the tasks that require root user](https://docs.aws.amazon.com/general/latest/gr/root-vs-iam.html#aws_tasks-that-require-root).
+
+This rule looks for attempts to log in to the AWS Management Console as the root user.
+
+#### Possible investigation steps
+
+- Investigate other alerts associated with the user account during the past 48 hours.
+- Examine whether this activity is common in the environment by looking for past occurrences on your logs.
+- Consider the source IP address and geolocation for the calling user who issued the command. Do they look normal for the
+  calling user?
+- Examine the commands, API calls, and data management actions performed by the account in the last 24 hours.
+- Contact the account owner and confirm whether they are aware of this activity.
+- If you suspect the account has been compromised, scope potentially compromised assets by tracking access to servers,
+services, and data accessed by the account in the last 24 hours.
+
+### False positive analysis
+
+- The alert can be dismissed if this operation is done under change management and approved according to the
+organization's policy to perform a task that needs this privilege level.
```

The last part of this sentence ("organization's policy to perform a task that needs this privilege level") seems a bit disconnected. Maybe switching "to" to "for" would link the two? Wdty?

```suggestion
organization's policy for performing a task that needs this privilege level.
```